sympy/solvers/ode.py

```diff
@@ -7882,7 +7886,7 @@ def _nonlinear_2eq_order1_type4(x, y, t, eq):
     arrives at a firs-order equation for determining `y` (resp., `x` ).
 
     """
-    C1, C2 = symbols('C1:3')
+    C1, C2, C3, C4 = get_numbered_constants(eq, num=4)
```

Comment:
I can just find C1 actually being used in this body. I haven't checked the other cases.